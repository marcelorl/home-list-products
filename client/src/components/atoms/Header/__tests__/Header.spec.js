import createTestContext from 'react-cosmos-test/enzyme'

import fixture from '../Header.fixture'

const { mount, getWrapper } = createTestContext({fixture})

beforeEach(mount)

describe('#Header atom', () => {
  it('renders component', () => {
    expect(getWrapper()).toMatchSnapshot()
  })

  it('should have a logo image', () => {
    expect(getWrapper('nav img')).toHaveLength(1)
  })
})
