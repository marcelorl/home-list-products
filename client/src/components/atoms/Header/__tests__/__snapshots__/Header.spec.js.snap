// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Header atom renders component 1`] = `
<Header>
  <Styled(Navbar)
    className="d-flex justify-content-center container-fluid px-0"
    color="faded"
    expand="lg"
    light={true}
  >
    <Navbar
      className="d-flex justify-content-center container-fluid px-0 sc-bdVaJa iQTWxT"
      color="faded"
      expand="lg"
      light={true}
      tag="nav"
    >
      <nav
        className="d-flex justify-content-center container-fluid px-0 sc-bdVaJa iQTWxT navbar navbar-expand-lg navbar-light bg-faded"
      >
        <styled.img
          alt="logo"
          src="logo.png"
        >
          <img
            alt="logo"
            className="sc-bwzfXH kXPULw"
            src="logo.png"
          />
        </styled.img>
      </nav>
    </Navbar>
  </Styled(Navbar)>
</Header>
`;
